crates/pathfinder/src/storage/schema/revision_0018.rs

```diff
@@ -0,0 +1,192 @@
+use anyhow::Context;
+use rusqlite::Transaction;
+
+/// Primary goal of this migration is to allow for StarkNet block forks.
+///
+/// This is achieved by:
+/// - switching `starknet_blocks` table PK from block number to hash
+/// - adding a `canonical_blocks` table to track the canonical chain
+/// - `starknet_events` now references `canonical_blocks` instead of `starknet_blocks`
+pub(crate) fn migrate(tx: &Transaction<'_>) -> anyhow::Result<()> {
+    use rusqlite::config::DbConfig::SQLITE_DBCONFIG_ENABLE_FKEY;
+    let fk_config = tx
+        .db_config(SQLITE_DBCONFIG_ENABLE_FKEY)
+        .context("Reading FK configuration")?;
+    tx.set_db_config(SQLITE_DBCONFIG_ENABLE_FKEY, false)
+        .context("Disabling foreign-key enforcement")?;
+    tracing::info!("hello");
```

Comment:
I think this line is just some left-over debugging thingy?

Reply:
Yes it is :(